docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst

```diff
@@ -156,4 +156,35 @@ Data Descriptions
 	  - YLD appendix
 	  - Disability weight of sequela with id {id}
 	  -
-	
\ No newline at end of file
+
+.. list-table:: Restrictions
```

Comment:
Shouldn't be in this PR.  If you're working on totally separate documents, remember to make all branches from master.

Reply:
oops; will make sure to do so in the future. thanks!